synapse/api/constants.py

```diff
@@ -119,3 +119,9 @@ class RoomVersions(object):
 
 ServerNoticeMsgType = "m.server_notice"
 ServerNoticeLimitReached = "m.server_notice.usage_limit_reached"
+
+
+# Allows for user type specific behaviour, if we'd had a crystal ball would
```

Comment:
could you make this a docstring rather than an inline comment?

(and use full stops rather than commas between sentences...)